iotdevice/mux.go

```diff
@@ -244,6 +251,9 @@ func (m *methodMux) Dispatch(method string, b []byte) (int, []byte, error) {
 	if err := json.Unmarshal(b, &v); err != nil {
 		return jsonErr(err)
 	}
+	if wildcard {
+		v["_method"] = method
```

Comment:
Can `v` be nil here? There's a check for that bellow.

Reply:
So technically if json.Unmarshal can't parse the JSON it returns an error, so this is actually another problem that today if you try to send a blank method it fails because it can't unmarhal the blank.  The lower check for nil is after the function call checking if the reply is nil from the callback pointer.